cmd/lakectl/cmd/tag.go

```diff
@@ -71,6 +71,18 @@ var tagCreateCmd = &cobra.Command{
 		client := getClient()
 		commitRef := args[1]
 		ctx := cmd.Context()
+		force, _ := cmd.Flags().GetBool("force")
+		if force {
+			// checking validity of the commitRef before deleting the old one
+			res, err := client.GetCommitWithResponse(ctx, tagURI.Repository, commitRef)
+			DieOnResponseError(res, err)
+
+			resp, err := client.DeleteTagWithResponse(ctx, tagURI.Repository, tagURI.Ref)
+			if err != nil && resp != nil && resp.JSON404 == nil {
```

Comment:
Wouldn't you want to die if there's an error but the response is nil?

Reply:
Right, done.